Tests/src/PublicAPITests/UserSystemTests.cpp

```diff
@@ -75,12 +71,47 @@ void LoadTestAccountCredentials()
 				 "<AlternativeLoginPassword>\n<SuperUserLoginEmail> <SuperUserLoginPassword>");
 	}
 
-	DefaultLoginEmail		 = _DefaultLoginEmail.c_str();
-	DefaultLoginPassword	 = _DefaultLoginPassword.c_str();
-	AlternativeLoginEmail	 = _AlternativeLoginEmail.c_str();
-	AlternativeLoginPassword = _AlternativeLoginPassword.c_str();
-	SuperUserLoginEmail		 = _SuperUserLoginEmail.c_str();
-	SuperUserLoginPassword	 = _SuperUserLoginPassword.c_str();
+	SuperUserLoginEmail	   = _SuperUserLoginEmail.c_str();
+	SuperUserLoginPassword = _SuperUserLoginPassword.c_str();
+}
+
+
+csp::systems::Profile CreateTestUser()
+{
+	auto& SystemsManager = csp::systems::SystemsManager::Get();
+	auto* UserSystem	 = SystemsManager.GetUserSystem();
+
+	const char* TestUserName	= "CSP-TEST-NAME";
+	const char* TestDisplayName = "CSP-TEST-DISPLAY";
+
+	char UniqueUserName[256];
+	SPRINTF(UniqueUserName, "%s-%s-%s", TestUserName, GetUniqueString().c_str(), GetUniqueString().c_str());
+
+	char UniqueEmail[256];
+	SPRINTF(UniqueEmail, GeneratedTestAccountEmailFormat, GetUniqueString().c_str());
+
+	// Create new user
+	auto [Result] = AWAIT_PRE(UserSystem,
+							  CreateUser,
+							  RequestPredicate,
+							  UniqueUserName,
+							  TestDisplayName,
+							  UniqueEmail,
+							  GeneratedTestAccountPassword,
+							  false,
+							  true,
+							  nullptr,
+							  nullptr);
+
+	EXPECT_EQ(Result.GetResultCode(), csp::systems::EResultCode::Success);
```

Comment:
[Consider] Some human friendly logging would go a long way here, folks won't know that they're baking assertions into their tests when they call `CreateTestUser`, and if this happens to fail, it'll speed up debugging flow a lot if you can see a "Failed to create temporary test user in `CreateTestUser`" message